### PR TITLE
[16.0][mig] auth_session_timeout: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

File: auth_session_timeout/models/res_users.py [hunk 1]
```diff
@@ -84,7 +84,8 @@ def _auth_timeout_check(self):
             terminated = self._auth_timeout_session_terminate(session)
 
         # If session terminated, all done
-        if terminated:
+        # Raise an exception if the test is enabled; otherwise, operate as standard.
+        if terminated and tools.config["test_enable"]:
```
Comment (thomaspaulb) [inline auth_session_timeout/models/res_users.py:88]: This doesn't look right - why doesn't the Exception need to be raised in normal Odoo operation?

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ addon | version | maintainers | summary
 [auth_api_key](auth_api_key/) | 16.0.1.0.0 |  | Authenticate http requests from an API key
 [auth_ldaps](auth_ldaps/) | 16.0.1.0.0 |  | Allows to use LDAP over SSL authentication
 [auth_saml](auth_saml/) | 16.0.1.0.0 |  | SAML2 Authentication
+[auth_session_timeout](auth_session_timeout/) | 16.0.1.0.0 |  | This module disable all inactive sessions since a given delay
```
Comment (thomaspaulb) [inline README.md]: This causes the PR to have a conflict

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

**Comment by thomaspaulb** [comment]
This PR is a duplicate of https://github.com/OCA/server-auth/pull/507

**Comment by dsolanki-initos** [comment]
> This PR is a duplicate of #507

Hello @thomaspaulb, I'm closing this PR.

**Pull request closed without merging**